magento/pwa-studio
# [PB] WYSIWYG Image optimization in PWA

## Description
WYSIWYG Image optimization in PWA

This PR enables WYSIWYG components to use the same sharp express back-end image optimization system as catalog images currently uses. It also opens the API for optimization options to the individual components 

## Related Issue
https://jira.corp.magento.com/browse/PB-5

## Related PR
https://github.com/magento-research/express-sharp/pull/4

## Verification Steps
1. Validate all existing Product images are still optimized
2. Verify row & column background images are resized and optimized
3. Verify optimization works for parallax images
